utmap-server/src/app/main/service/eventService.py

```diff
@@ -13,33 +13,25 @@ def addEvent(data):
 
```

Comment:
isValidEventInput() does not work as intended. Due to how Python works, you need to do `'name' in keyList and 'organizer' in keyList and ...`  
`('name' and 'organizer' and 'startTime')` will give you 'startTime', so `('name' and 'organizer' and 'startTime') in keyList` will turn into `'startTime' in keyList`.

Reply:
I see.  I'll fix that.